samples/hello++/main.cpp

```diff
@@ -1,27 +1,21 @@
 #include <hal/debug.h>
 #include <hal/video.h>
+#include <string>
+#include <vector>
 #include <windows.h>
 
-class NumberClass{
-private:
-  int hiddenValue = 0;
-public:
-  NumberClass() {hiddenValue = 5;};
-  NumberClass(int v) : hiddenValue(v) {};
-  int getValue() { return hiddenValue; };
-};
-
 int main(void) {
   XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
+  std::vector<std::string> words;
```

Comment:
I'd add an empty line before this, to add some structure to the code. Other instances, too.